src/feature/Client/components/Filters/Filter.scss

```
@@ -0,0 +1,49 @@
+@import "assets/styles/variables.scss";
+.active{
+    color: $blue;
+}
+
+.categoryItem{
```

Check lại cả file nhé
```suggestion
.active {
    color: $blue;
}

.categoryItem {
```